operations/patch_util.go

```diff
@@ -181,6 +182,10 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 		grip.Warningf("warning - failed to set default tasks: %v\n", err)
 	}
 
+	if p.WorkingTree || conf.FindDefaultWorkingTree(p.Project) {
```

Comment:
Can we print a message about the include-working flag if not passed, and remove it in a month? That way users who don't pay attention to usual channels of communication will know about the change to behavior

Reply:
done.